pylint/lint/pylinter.py

```diff
@@ -600,7 +600,7 @@ def should_analyze_file(modname: str, path: str, is_argument: bool = False) -> b
         """
         if is_argument:
             return True
-        return path.endswith(".py")
+        return path.endswith(("py", ".pyi"))
```

Comment:
```suggestion
        return path.endswith((".py", ".pyi"))
```

Reply:
Thanks for catching an error I made in haste!